packer.json

```diff
@@ -2,6 +2,19 @@
   "builders": [
     {
       "type": "amazon-ebs",
+      "name": "empire-eu-west-1",
+      "access_key": "",
+      "secret_key": "",
+      "region": "eu-west-1",
+      "source_ami": "ami-47a23a30",
```

Comment:
This is a fairly old AMI - may want to use a more modern one.  The most recent: ami-92401ce5